packages/dbadmin/src/controllers/DBAdminApi.ts

```diff
@@ -55,7 +55,12 @@ export class DBAdminApiImpl implements DBAdminApi {
     createDatabase = async(request: e.Request, response: e.Response) => {
         logger.requestLog(` * createDatabase request received, with body of ${request.headers["content-length"]} bytes`)
         const result = await this.ctx.dbAdminApiWorker.createDatabase(CREATE_DATABASE_SQL)
-        lionwebResponse(response, result.status, {
+        // lionwebResponse(response, result.status, {
```

Comment:
Can we remove this code or is this kept in a comment for a reason?

Reply:
Can be removed.